src/utils/navigation-actions.js

```diff
@@ -104,8 +104,22 @@ const navigationActions = [
     },
     requiredInput: ['websiteUrl'],
   },
+  {
+    translation: 'Object.ActionButton.SelectOdagApp',
+    value: 'odagLink',
+    navigationCall:
+      async ({ app, senseNavigation, odagLink, element }) => {
+        if (typeof senseNavigation.openOdagPopup === 'function' && odagLink) {
+          await senseNavigation.openOdagPopup(app, odagLink, element);
```

Comment:
How will this work outside the client? E.g. embedded sheet, embedded button, mashups...

Reply:
I don't think the Odag will work outside the sense-client now. So this will not work either. This change is specifically to create Odag from button object.

Reply:
Understand that, but you can create a button in sense client (and configure) and then have the button outside by embedding a sheet for instance. You need to test these scenarios

Reply:
In embedded sheet mode, the button is trying to make backend call and couldn't fetch data and failing there which is expected.